package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "changelog-maker",
-  "version": "1.2.4",
+  "version": "1.2.5",
```

Comment:
one last nit: please don't version bump, I use `npm version` to do this and it signs releases git tags for me in the process.